packages/toolkit/src/utils/ipfs.util.ts

```diff
@@ -1,22 +1,21 @@
 import axios from 'axios'
-import type { ImportCandidate } from 'ipfs-core-types/src/utils'
-import { create, type IPFSHTTPClient } from 'ipfs-http-client'
+import { create, type IPFSHTTPClient } from 'kubo-rpc-client'
 
 import { toolkitConfig } from '@/globals'
 
 const PREFIX = 'ipfs://'
 
 export class IpfsUtil<T> {
   private _rawData?: T
-  private _path?: string
-  private _paths = [] as string[]
+  private _cid?: string
```

Comment:
id `add` and `addAll` returns path, should we use `this.path` instead of `cid`?

Reply:
Yes, we can rename getters because we return a cid paths, but in class we're using cids, not cid paths.

Reply:
add comment

Reply:
done